### PR TITLE
Create overview.md

### PR DESCRIPTION
**Pull request by fabioflx**
I'm going to support Buffalo, starting with an Italian translation.

### CODE REVIEW

File: templates/it/docs/overview.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<% seoDescription("Buffalo framework overview") %>
+<% seoKeywords(["buffalo", "framework", "overview", "go", "golang", "mux", "bootstrap", "jquery"]) %>
+
+<%= h1("Overview") %>
```
Comment (stanislas-m) [inline templates/it/docs/overview.md:4]: This is the title of the page, can you translate it too? :)

File: templates/it/docs/overview.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+### Webpack
+
+[Webpack](https://webpack.js.org/) è un famoso gestore di pacchetti per Javascript. Si prenderà cura dei tuoi Javascript, CSS, immagini e asset statici.
+
+Webpack è configurato di default per versionare e minimizzare i tuoi asset.
+
+## Prossimi passi
+
+* [Installazione](/en/docs/getting-started/installation) - Installa Buffalo!
```
Comment (stanislas-m) [inline templates/it/docs/overview.md:52]:
```suggestion
* [Installazione](/it/docs/getting-started/installation) - Installa Buffalo!
```

File: templates/it/docs/overview.md (hunk 1 above)
Comment (fabioflx) [reply templates/it/docs/overview.md]: Sure. Did it

**Comment by stanislas-m** [comment]
Grazie !

**Pull request merged**